recipes/discretizer/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "discretizer" %}
+{% set version = "0.4.3" %}
+{% set sha256 = "de557f087061ea9a0e807e63423d3b325985b262524fdaacdb583749b1eded7a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
```

Comment:
So `fn` is just the name the file is saved as by `conda-build` not the name of where it was gotten. Thus it shouldn't matter if it were GitHub, GitLab, or some storage site. Nor should it matter what they chose to name the file.